cosmos/providers/dbt/core/operators/local.py

```diff
@@ -128,6 +130,29 @@ def run_command(
                 tmp_project_dir,
             )
 
+            # get the profile name from the dbt_project.yml file
+            dbt_project_path = os.path.join(tmp_project_dir, "dbt_project.yml")
+            with open(dbt_project_path, encoding="utf-8") as f:
+                dbt_project = yaml.safe_load(f)
```

Comment:
It would be great if we could add tests to cover this part of the code.

Reply:
Not sure I understand what you want tested here - opening the file? I added another check to ensure the file exists, let me know if you still want more.